README.md

```diff
@@ -149,18 +148,13 @@ $ npm install --global flow-bin
 
 There are a few customizations that are supported by this extension. They can be added to your `.vscode/settings.json` if you need them.
 
-For using another simulator device which is different to the default `iPhone 5`:
+For using custom run arguments for `react-native run-<platform>`:
+* **Note:** Override all other configuration params
 
 ```
 {
-  "react-native.ios.launchTarget": {
-    "simulator" : "iPad Pro (9.7 inch)",
-    "device": "Max's iPhone"
-  },
-  "react-native.android.launchTarget": {
-    "simulator" : "emulator-5556",
-    "device": "emulator-5556"
-  }
+  "react-native.android.simulator.runArgs": ["--appFolder, "/Users/test/AwesomeProject/android/app", "--deviceId", "emulator-5555"],
```

Comment:
Can we expose it as `react-native.android.runArgs.simulator` rather than `react-native.android.simulator.runArgs` similar to former 'launchTarget' option?

Also IMHO `runArgs` doesn't sound well - it's convenient to use in code but in configuration file shortening names is not necessary and IMO decreases readability of config file